ports/concurrencpp/portfile.cmake

```
@@ -0,0 +1,23 @@
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO David-Haim/concurrencpp
+  REF v.0.1.2
+  SHA512 48e3acca73c7b06940d41e84334cec56e7657c45f862d53e8e4ca1fb76b792c5b136ce4eeaf8afb8db95e746442fb45031c2c695ce60fcbe92ce91e5e1e80b25

```

```suggestion
  REF ffe6ec162dc32d23097a34b13ab0b3ed4a37128b
  SHA512 1a02eca8519c4668c90fd02570965d90cd20458ded8e9615fdae251dcddda644847507720ebf1c6d528f7f034196ae83856bee34bc6a20227642d807065839f4
```
This fixes the macos build

---

Great, thanks! But shouldn't we rather wait until there is a new stable version released with these fixes? I am not sure what else is in the `develop` branch, there might be unstable changes or other bugs.

---

Yeah maybe we should ask for a new release. Or only test if this really fixes the build. Alternatively you could apply the patch that fixes the build manually

---

Let's wait for the next release to include those fixes.